core/logger/logger.go

```diff
@@ -16,7 +16,68 @@ import (
 
 // Logger is the main interface of this package.
 // It implements uber/zap's SugaredLogger interface and adds conditional logging helpers.
-type Logger struct {
+type Logger interface {
+	// Write logs a message at the Info level and returns the length
+	// of the given bytes.
+	Write(p []byte) (n int, err error) // io.Writer
+
+	// Named creates a new named logger with the given name
+	Named(name string) Logger
+	// With creates a new logger with the given arguments
+	With(args ...interface{}) Logger
+	// WithError creates a new logger with the given error
+	WithError(err error) Logger
+	// WithCallerSkip creates a new logger with the number of callers skipped by
+	// caller annotation increased by add.
+	WithCallerSkip(add int) Logger
```

Comment:
Two log wrappers: `ormLogWrapper` and `ocrLogger`. It might be easy to move them both in to the logger package so they can use the internals to do this directly via zap instead.

Reply:
Moved and made this method unexported.